Formula/chapel.rb

```diff
@@ -16,26 +15,30 @@ class Chapel < Formula
   end
 
   depends_on "gmp"
-  depends_on "llvm@11"
+  depends_on "llvm"
   depends_on "python@3.10"
 
-  # Include changes for Python 3.10 migration
-  # They are already merged upstream after 1.25.1
-  # TODO: Remove this in the next release
-  patch :DATA
+  # LLVM is built with gcc11 and we will fail on linux with gcc version 5.xx
+  fails_with gcc: "5"
 
   def install
     libexec.install Dir["*"]
     # Chapel uses this ENV to work out where to install.
     ENV["CHPL_HOME"] = libexec
     ENV["CHPL_GMP"] = "system"
+    # This enables a workaround for
+    #   https://github.com/llvm/llvm-project/issues/54438
+    ENV["CHPL_HOST_USE_SYSTEM_LIBCXX"] = "yes"
 
     # Must be built from within CHPL_HOME to prevent build bugs.
     # https://github.com/Homebrew/legacy-homebrew/pull/35166
     cd libexec do
       system "echo CHPL_RE2=bundled > chplconfig"
       system "echo CHPL_GMP=system >> chplconfig"
-      system "echo CHPL_LLVM_CONFIG=#{HOMEBREW_PREFIX}/opt/llvm@11/bin/llvm-config >> chplconfig"
+      system "echo CHPL_LLVM_CONFIG=#{HOMEBREW_PREFIX}/opt/llvm@13/bin/llvm-config >> chplconfig"
+      # don't try to set CHPL_LLVM_GCC_PREFIX since the llvm@13
+      # package should be configured to use a reasonable GCC
+      system 'echo CHPL_LLVM_GCC_PREFIX="none" >> chplconfig'
```

Comment:
This could be easier like:
```
(libexec/"chplconfig").write <<~EOS
  CHPL_RE2=bundled
  CHPL_GMP=system
EOS
```

Reply:
Thanks for that and I love easier :)   Will get that in for our production build...